### DISCUSSION TITLE
O2M default filter and sort

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#15932] [Feature Requests]
A few weeks ago the o2m interface introduced a new `table` layout including a search, sort and filter option. 
It would be great if we could add a filter and sort as default and maybe even save some views same as with the collection. This would allow users to build kind of different views on their relations.

Example: Only display related items with the status "published" by default, allow to change the filter if we also want to see 'drafts'

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
